mmdet/models/backbones/csp_darknet.py

```diff
@@ -0,0 +1,269 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, DepthwiseSeparableConvModule
+from mmcv.runner import BaseModule
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from ..builder import BACKBONES
+from ..utils import CSPLayer
+
+
+class Focus(nn.Module):
```

Comment:
Do we need to inherit BaseModule and add init_cfg?

Reply:
If use BaseModule here, the init_cfg in the config file will be complicated. Same as using mmcv ModuleList and Sequential